slack_bolt/middleware/function_token/function_listener_token.py

```diff
@@ -0,0 +1,24 @@
+from typing import Callable
+
+from slack_bolt.request import BoltRequest
+from slack_bolt.response import BoltResponse
+from slack_bolt.middleware.middleware import Middleware
+
+
+class FunctionToken(Middleware):  # type: ignore
```

Comment:
How about making the name a bit clearer?
```suggestion
class AttachingFunctionToken(Middleware):  # type: ignore
```

Reply:
Love this name, was struggling to find something meaningful 🥇